code/modules/mining/mine_items.dm

```diff
@@ -292,351 +292,3 @@
 		colorindex = 0
 	icon_state = "mobcap[colorindex]"
 	update_icon()
-
-/*****************************Survival Pod********************************/
```

Comment:
Any changes made to this region?

Reply:
I added the NanoMed, switched the survival pod to be loaded by default with an empty subtype (to be consistent with /tg/), replaced the telecrystal artifact with a stack of twenty (which are fake anyway) and fixed some formatting. That also happens to fix a bug where you can't change the type of the smartfridge circuit board.